src/summariser.py

```diff
@@ -36,15 +66,19 @@ def cohere_summarise(message: str) -> NonStreamedChatResponse:
 
 
 def summarise_webpage(url: str) -> str:
-    # html_text = get_html_from_url(url)
-    cohere_response: NonStreamedChatResponse = cohere_summarise(url)
+    html_text = langchain_webcontent(url)
+    cohere_response: NonStreamedChatResponse = cohere_summarise(html_text)
     text: str = cohere_response.text
     return text
 
 
 if __name__ == "__main__":
-    url = "https://samharris.substack.com/p/october-7?utm_source=substack&publication_id=471923&post_id=145856573&utm_medium=email&utm_content=share&utm_campaign=email-share&triggerShare=true&isFreemail=true&r=1s4h2t&triedRedirect=true"
     url = "https://conversationswithtyler.com/episodes/nassim-nicholas-taleb-and-bryan-caplan/"
+    url = "https://samharris.substack.com/p/october-7?utm_source=substack&publication_id=471923&post_id=145856573&utm_medium=email&utm_content=share&utm_campaign=email-share&triggerShare=true&isFreemail=true&r=1s4h2t&triedRedirect=true"
+    url = "https://www.restorationbulletin.com/p/standing-on-a-slackline-giorgia-meloni"
+    url = "http://www.restorationbulletin.com/p/standing-on-a-slackline-giorgia-meloni"
+    url = "https://www.bitsaboutmoney.com/archive/working-title-insurance/"
+    url = "https://www.complexsystemspodcast.com/episodes/teaching-trading-ricki-heicklen/"
 
     incoming_msg = f"@summarise {url}"
     summary = summarise_webpage(incoming_msg)
```

Comment:
Does this mean the script can only summarise the complex systems podcast? Because it's hardcoded? And where does this value go, does it render?

Reply:
Like you said it's for testing! So probably a test file